packages/sdk/src/evm/contracts/prebuilt-implementations/marketplace.ts

```diff
@@ -36,7 +36,7 @@ import invariant from "tiny-invariant";
  * import { ThirdwebSDK } from "@thirdweb-dev/sdk";
  *
  * const sdk = new ThirdwebSDK("{{chainName}}");
- * const contract = sdk.getMarketplace("{{contract_address}}");
+ * const { contract } = sdk.getContract("{{contract_address}}", "marketplace");
```

Comment:
Uuuh we destructuring now? Looks wrong

Reply:
wait yeah this is wrong the fuck am I doing lol